src/formatMarkup.js

```diff
@@ -166,36 +243,50 @@ export const diffableFormatter = function (markup) {
         !tagIsEscapabelRawTextElement
       )
     );
+
+    // InnerText
+    if (defaultTreeAdapter.isTextNode(node)) {
+      if (areAllTagsAreWhiteSpaceDependent) {
+        return node.value;
+      }
+      if (node.value.trim()) {
+        let nodeValue = node.value;
+        if (options.escapeInnerText) {
+          nodeValue = escapeHtml(nodeValue);
+        }
+        if (tagIsWhitespaceDependent) {
+          return nodeValue;
+        } else {
+          return '\n' + '  '.repeat(indent) + nodeValue.trim();
+        }
+      }
+      return '';
+    }
+
+    // <!-- Comments -->
+    if (defaultTreeAdapter.isCommentNode(node)) {
```

Comment:
I was like "oh, I guess my simple string check isn't the right way to do this. I wonder what advance AST check this is doing.

* https://github.com/inikulin/parse5/blob/master/packages/parse5/lib/tree-adapters/default.ts#L291

It's literally the same thing. There is an argument that this safe guards us from upstream changes. But if they are changing the way the AST is generated to this degree we're going to have bigger issues.

I don't want to have to go look up this source code every time I want to know what this is doing.

```suggestion
    if (node.nodeName === '#comment') {
```